ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,199 @@
+
+
+When clicking an action, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+
+The action settings side bar is located to the right of the workspace and allows to define the action settings. The following fields are displayed:
+
+  - (Action label and name)  
+  The action label is an action identification tag and can be defined by user. The action name is a system wide predefined denomination of the activity to be performed and is read-only.
+
+  - *Key*  
+   Action key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
+
+  - *Queue type*  
+  Click the drop-down list to select the applicable queue type. For detailed information about queue types, see [Configure the queue types](/ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md).
+
+  - *Task event*  
+  Click the drop-down list to select the applicable task event.
+
+  > [Info] This option is linked to the *Tasks* module. It is only displayed if the *Tasks* module is installed and the user has the necessary rights. The *ActindoWorkflow Process Execution Failed* task is preconfigured by default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
+
+  - *Configuration*  
+  Depending on the action, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the action configuration may require basic programming knowledge.
+
+  > [Info] Most actions are preconfigured and do not include configuration options. In this case, the *Configuration* section is not displayed.
+
+  - *Static inputs*  
+  All unlinked inputs are listed in this section. Static inputs can be used to configure static values for certain inputs. They can be defined by the user clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button. Only valid JSON values are accepted. When edited, static inputs display a small arc without a place in front of them in the workflow diagram.
+
+  > [Info] Normally, all tokens for the action execution are taken from the places connected to its input ports. However, static values can be configured for certain inputs. This way, if an action has configured a static input on one or more of its input ports, it will behave as if there was a place connected to that input port that always contains one token (containing whatever value is statically configured). Nevertheless, it is still necessary to have **at least one** "actual" place with an "actual" token in order to make that action executable.
```

Comment:
```suggestion
  > [Info] Normally, all tokens for the action execution are taken from the places connected to its input ports. However, static values can be configured for certain inputs. This way, if an action has configured a static input on one or more of its input ports, it will behave as if there was a place connected to that input port that always contains one token (containing whatever value is statically configured). Nevertheless, it is still necessary to have at least one "actual" place with an "actual" token in order to make that action executable.
```
s.o.
was meinst du mit "actual" ?